dashboard/pages/hit_validation/callbacks.py

```diff
@@ -270,6 +300,8 @@ def on_selected_compound_changed(
             ]
         ),
         "graph": graph,
+        "top": entry["TOP"],
+        "bottom": entry["BOTTOM"],
```

Comment:
I'd set the precision to 3-4 decimal places here

Reply:
set it to 5 for consistency